src/internal.c

```diff
@@ -10500,6 +10500,7 @@ int CheckHostName(DecodedCert* dCert, const char *domainName, size_t domainNameL
         ret = 0;
     }
 
+#ifndef ONLY_ALT_NAME_VERIFICATION
```

Comment:
Can we rename this to something like `WOLFSSL_HOSTNAME_VERIFY_ALT_NAME_ONLY`? Please add description of build option at top of internal.c with others.